src/main/java/nl/novi/behuiswerkles11/dtos/TelevisionRequestDto.java

```diff
@@ -0,0 +1,148 @@
+package nl.novi.behuiswerkles11.dtos;
+
+// input van de repository (http request)
+
+import jakarta.validation.constraints.NotEmpty;
+import org.antlr.v4.runtime.misc.NotNull;
+
+// TODO Vraag: Wat is het verschil tussen @NotNull, @NotEmpty en @NotBlank? En welke heeft de voorkeur?
```

Comment:
Deze annotaties hebben een oplopende strictheid van wat een ingevulde String waarde is (of wanneer het dus niet een ingevulde String waarde is). 
- De minst stricte is NotNull. Hier wordt alleen gecheckt of er iets is ingevuld. "" , "  " en "iets" zijn geldige waardes. Wanneer je deze waarde leeg laat in je JSON request, dan wordt het wel afgekeurd.
- Daarna komt NotEmpty. Deze checkt dat er iets ingevuld moet zijn. "" is hier geen geldige waarde, maar "  " en "iets" wel
- De strictste is NotBlank. Hier moet er daadwerkelijk iets worden ingevuld in de String. Ook enkel witruimte is niet goed. "" en "  " worden afgekeurd, maar "iets" is wel goed.